plumbing/api.go

```diff
@@ -181,6 +182,11 @@ func (api *API) NetworkGetPeerID() peer.ID {
 	return api.network.GetPeerID()
 }
 
+// RouterFindProvidersAsync issues a findProviders query to the router.
+func (api *API) RouterFindProvidersAsync(ctx context.Context, key cid.Cid, count int) <-chan pstore.PeerInfo {
```

Comment:
NetworkFindProvidersAsync seemed unnecessarily general but input welcome on the name.

Reply:
`DHTFindProvidersAsync`? "Router" doesn't mean anything to me as a filecoin user.

Reply:
Hmm `Router` shows up as a dependency of the node and now the plumbing so it should mean something right?  The thing is the thingy that calls `FindProvidersAsync` isn't always a DHT, sometimes its an offline router that always returns nothing.